app/shared/resource-calendar/_resource-calendar.scss

```diff
@@ -2,190 +2,218 @@
   $busy-color: $yellow;
   $booked-color: $red;
-  display: block;
-  max-width: 400px;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
   margin: 30px auto;
-  position: relative;
 
   &__icon {
     width: 22px;
   }
-
-  .form-group {
-    margin-bottom: 0;
-  }
-  .form-control[disabled] {
-    background-color: transparent;
-    border-color: $black;
-    color: $black;
-    cursor: default;
-    transition: all 0s ease 0s;
-  }
-
-  .input-group-addon {
-    background-color: transparent;
-  }
-
-  .glyphicon-triangle-bottom {
-    font-size: 10px;
-  }
-
-  .form-control[disabled] {
-    border-left-width: 0;
-    border-right-width: 0;
-  }
-
-  .DayPicker {
+  
+  &__week-button {
+    background: none;
+    border: none;
     padding: 0;
-    overflow-x: hidden;
-
-    &:focus {
-      outline: none;
-    }
-  }
-
-  .DayPicker-NavBar {
-    padding: 0.75rem 0.5rem;
-  }
-
-  .DayPicker-Caption {
-    height: auto;
-    font-weight: 600;
-    line-height: 3rem;
-  }
-
-  .DayPicker-Month {
     margin: 0;
-    border-collapse: separate;
-  }
-
-  .DayPicker-Day {
-    border: 2px solid $white;
-    padding: 1.2rem;
-    margin: 1px;
-    line-height: 1rem;
-    color: $black;
-    background-color: $available-color;
+    width: 40px;
+    height: 40px;
+    
+    &--prev {
+      @include icon-angle-left($black);
+    }
 
-    &:focus {
-      outline: none;
+    &--next {
+      @include icon-angle-right($black);
     }
+  }
 
-    &--disabled {
-      cursor: pointer;
-      text-decoration: line-through;
-      background-color: $medium-gray;
-      color: $dark-gray;
+  .app-ResourceCalendar__wrapper {
+    display: block;
+    max-width: 400px;
+    flex-grow: 1;
+    position: relative;
+    .form-group {
+      margin-bottom: 0;
     }
-    &--available:not(.DayPicker-Day--disabled) {
-      background-color: $copper;
-      &.DayPicker-Day--selected {
-        background-color: $available-color;
+    .form-control[disabled] {
+      background-color: transparent;
+      border-color: $black;
+      color: $black;
+      cursor: default;
+      transition: all 0s ease 0s;
+    }
+  
+    .input-group-addon {
+      background-color: transparent;
+    }
+  
+    .glyphicon-triangle-bottom {
+      font-size: 10px;
+    }
+  
+    .form-control[disabled] {
+      border-left-width: 0;
+      border-right-width: 0;
+    }
+  
+    .DayPicker {
+      padding: 0;
+      overflow-x: hidden;
+  
+      &:focus {
+        outline: none;
       }
     }
-    &--busy:not(.DayPicker-Day--disabled) {
-      background-color: $busy-color;
-      &.DayPicker-Day--selected {
+  
+    .DayPicker-NavBar {
+      padding: 0.75rem 0.5rem;
+    }
+  
+    .DayPicker-Caption {
+      height: auto;
+      font-weight: 600;
+      line-height: 3rem;
+    }
+  
+    .DayPicker-Month {
+      margin: 0;
+      border-collapse: separate;
+    }
+  
+    .DayPicker-Day {
+      border: 2px solid $white;
+      padding: 1.2rem;
```

Comment:
IMO some  custom value like this should be store in local sass variable as much as you can

Reply:
I agree I prefer extracting all of them to `_variables` module though, which I will take care of in a new ticket.